example/pom.xml

```diff
@@ -30,17 +28,64 @@
 			<groupId>org.jetbrains.kotlin</groupId>
 			<artifactId>kotlin-stdlib</artifactId>
 		</dependency>
-
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-jdbc</artifactId>
+		</dependency>
 		<dependency>
 			<groupId>org.postgresql</groupId>
 			<artifactId>postgresql</artifactId>
 			<scope>runtime</scope>
 		</dependency>
+
+		<!--
+		For running testcontainers outside tests. And without including it in the distributed fat JAR.
```

Comment:
Need to review this statement. I might have found a way around it through the use of LocalDevTestContainersConfig in the test directory. 

In other words: maybe I should narrow the scope of this dependency to "test"